lib/ui/submission/survey_submission_success_screen.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:lottie/lottie.dart';
+import 'package:survey_flutter_ic/extension/context_extension.dart';
+import 'package:survey_flutter_ic/theme/dimens.dart';
+
+const _animationDurationInMilliseconds = 2000;
+
+class SurveySubmissionSuccessScreen extends StatefulWidget {
+  const SurveySubmissionSuccessScreen({Key? key}) : super(key: key);
+
+  @override
+  State<SurveySubmissionSuccessScreen> createState() =>
+      _SurveySubmissionSuccessScreenState();
+}
+
+class _SurveySubmissionSuccessScreenState
+    extends State<SurveySubmissionSuccessScreen> with TickerProviderStateMixin {
+  late final AnimationController _controller;
+  bool _messageVisible = false;
+
+  @override
+  void initState() {
+    _controller = AnimationController(
+      vsync: this,
+      duration: const Duration(milliseconds: _animationDurationInMilliseconds),
+    );
+    Future.delayed(
+        const Duration(milliseconds: _animationDurationInMilliseconds), () {
+      setState(() {
+        _messageVisible = true;
+      });
+    });
```

Comment:
Instead of triggering a delay for showing the message, I think it's better to show the message after loading the animation. So we can get rid of this `Future.delayed`

Reply:
@manh-t Oh nice 👍 We could combine the animation flow ❤️